Atos20/Rancid-Tomatillos
# Feature/homepage

### PR Template:

#### What’s this PR do?  
This PR adds the card field and a place holder card to add styles to for later.
 
#### Where should the reviewer start?  
Homepage (or HomePage).
 
#### How should this be manually tested?  
Run ```npm start``` This will bring you to the local host where you can check out the basic JSX layout
 
#### Any background context you want to provide?  
We got confused on naming techniques so there might be a few extra commits that never really needed to exist.
 
#### What are the relevant tickets?  
#6 
#4 
 
#### Screenshots (if appropriate) 
n/a 

#### Questions: 
n/a
